http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/db/KubernetesServiceDbQueries.scala

```diff
@@ -76,6 +77,17 @@ object KubernetesServiceDbQueries {
       }
     } yield eitherClusterOrError
 
+  // This method is used to retrieve the namespace and nodepool that belong to a user
+  // It assumes these values are the same for a user within a project across all apps
+  // It will only ever return the first app found for this reason
+  def getAppForUser(googleProject: GoogleProject, creator: WorkbenchEmail, labelFilter: LabelMap = Map())(
```

Comment:
`NODEPOOL` has a `creator` column, right? I'm wondering if a `getNodepoolForUser` method would be a more direct way to do it this.